api/chaos/experimentation/v1/experimentation.proto

```diff
@@ -109,27 +119,32 @@ service ExperimentsAPI {
     };
     option (clutch.api.v1.action).type = CREATE;
   }
+  // Cancel a running experiment run. The operation returns success even if a given experiment run has been
+  // already completed or canceled and the operation does not have any effect.
   rpc CancelExperimentRun(CancelExperimentRunRequest) returns (CancelExperimentRunResponse) {
     option (google.api.http) = {
       post : "/v1/chaos/experimentation/cancelExperimentRun"
       body : "*"
     };
     option (clutch.api.v1.action).type = DELETE;
   }
+  // Get the list of experiments fulfilling the specified criteria. See `GetExperimentsRequests` for more details.
   rpc GetExperiments(GetExperimentsRequest) returns (GetExperimentsResponse) {
     option (google.api.http) = {
       post : "/v1/chaos/experimentation/getExperiments"
       body : "*"
     };
     option (clutch.api.v1.action).type = READ;
   }
+  // Get the list of experiments in the form that's optimized for displaying to the end user.
   rpc GetListView(GetListViewRequest) returns (GetListViewResponse) {
     option (google.api.http) = {
       post : "/v1/chaos/experimentation/getListView"
       body : "*"
     };
     option (clutch.api.v1.action).type = READ;
   }
+  // Fetch the list of properties in the format that's optimized for displaying to the end user.
```

Comment:
Just a note but it's a bit odd to have endpoints that are tailored for rendering and for general lookups without any naming differences. Seems to be conflating a regular CRUD API with generating UI elements. 

Maybe these should be separated in some way, e.g. name prefix/suffix, different API, etc.?

Out of scope of this PR, just a thought

Reply:
Ah, that makes sense - I will follow up on this